OneCommunityGlobal/HighestGoodNetworkApp
# completed copy button for weekly summaries and user profile

# Description
Created a copy button on the user's weekly summary and the weekly summary branch. 
Recreated this PR as the original one, won't load the weekly summaries, and had several changes, which came from the navbar disappearing. 

<img width="855" alt="Screenshot 2023-10-12 at 8 35 43 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/22454953/67208806-90e6-4d55-aec7-1831ff7178c1">


Please reviewing the original PR to test this one. 

Added a copy icon to user's profile component. When viewing weekly summaries admins can quickly copy and paste their team members summaries to speed up their managerial tasks.
Jae also suggested to add the icon to the weekly reports summary to view everyone's weekly summaries to counter-act the current bug. This way admins can copy weekly summaries there as well.


## Related PRS (if any):

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1166

## How to test:
check into current branch
do npm install and ... to run this PR locally
log as admin user or owner.
go to dashboard→ view profile → show weekly summary → select a team member.
View their weekly summaries and click on the copy button to copy their summary to your clipboard.
Verify clipboard has the exact text copied to your clipboard.
Repeat from steps 3 but navigate to dashboard→ reports → weekly summary reports → select a team member.
View their weekly summaries and click on the copy button to copy their summary to your clipboard.

## Screenshots or videos of changes:

## Note:
Include the information the reviewers need to know.
